bin/setup-athena-tables.ts

```diff
@@ -0,0 +1,47 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+import 'dotenv/config'
+
+import AthenaClient from '../server/data/athenaClient'
+import S3Client from '../server/data/s3Client'
+
+const table = 'sample'
+const sampleCsvPath = path.resolve(process.cwd(), 'data', 'seattle-weather.csv')
+const databaseS3Path = 'athena/sample/' // NB: must end in slash
+const sampleCsvS3Path = `${databaseS3Path}data.csv`
+
+function getEnvironmentVariable(name: string) {
+  if (process.env[name]) {
+    return process.env[name]
+  }
+  throw new Error(`Missing env var ${name}`)
+}
+
+async function main() {
+  const s3Client = new S3Client({
+    accessKeyId: getEnvironmentVariable('S3_ACCESS_KEY_ID'),
+    secretAccessKey: getEnvironmentVariable('S3_SECRET_ACCESS_KEY'),
+    bucket: getEnvironmentVariable('S3_BUCKET_NAME'),
+  })
+  process.stderr.write(`Uploading “${path.basename(sampleCsvPath)}” to “${sampleCsvS3Path}” is S3 bucket…\n`)
+  await s3Client.putObject(sampleCsvS3Path, fs.createReadStream(sampleCsvPath))
+
+  const athenaClient = new AthenaClient({
+    accessKeyId: getEnvironmentVariable('ATHENA_ACCESS_KEY_ID'),
```

Comment:
if you move these values into `server/config` (which I think it's worth as I'd imagine they'd be used in few places) you can:
1. avoid this whole `getEnvironmentVariable()` utility function
2. just import/use `server/config` here
3. pass `config.athena` to this client directly

Basically you can get rid of quite few lines of code here and make it even simpler ☺️

Reply:
e.g.

```
import config from `../server/config`

// no need for getEnvironmentVariable(), this logic is already in `server/config`

async function main() {
  const s3Client = new S3Client(config.athena)

  // ...
}
```

Reply:
i was planning on doing that in the next PR once i add the athena client to the visualisation service. to me it's not really clear if athena will _ever_ be needed within the app. i see it more as a periodic data processor in the background because of how it's execution is asynchronous